### DISCUSSION TITLE
Distillation Columns in Dynamics Mode

### DISCUSSION THREAD

**Discussion by svidille** [DanWBR/dwsim#572] [General]
Hi, Dan

First of all, I will like to say that I admire your work and contributions to the Chemical Engineering world. Congratulations. DWSIM is a great surprise for me.

I am testing dynamics mode right now and I noticed that there some exclamation marks information that columns are supported in this mode. Is there any prevision for when the columns will be available in dynamics mode? 

Thanks,
Best wishes,
Simone

**Comment by DanWBR** [comment]
it is not in the plans at the moment, @svidille.
